CMakeLists.txt

```diff
@@ -504,6 +515,15 @@ if(SENTRY_WITH_LIBUNWINDSTACK)
 	endif()
 endif()
 
+if(SENTRY_WITH_LIBUNWIND)
+	if(LINUX)
+		find_path(LIBUNWIND_INCLUDE_DIR libunwind.h PATH_SUFFIXES libunwind REQUIRED)
+		find_library(LIBUNWIND_LIBRARY unwind REQUIRED)
```

Comment:
Note: manual find instead of pkg-config because `llvm-libunwind` doesn't ship a .pc file